### DISCUSSION TITLE
react-aria-component tooltip: please provide option to trigger tooltip open on long button press and hold

### DISCUSSION THREAD

**Discussion by rolanday** [adobe/react-spectrum#4884] [Ideas]
Please consider an option for tooltip to show after set period of time pressing and holding a trigger button. Other frameworks like radix support this behavior, as it is intuitive discovery mechanism -- esp. on touch enabled devices, but also for mouse.

Thanks :-)

**Comment by devongovett** [comment]
See #1136. I think the issue is that long press gestures often conflict with other gestures like context menus (native or otherwise). Usually we just recommend avoiding tooltips if they are important, and using a popover on an adjacent element instead. In Spectrum we have the [ContextualHelp](https://react-spectrum.adobe.com/react-spectrum/ContextualHelp.html#contextualhelp) component for this, but you could make something like it yourself with the Popover component in RAC.

**Reply by rolanday** [reply]
Thanks Devon. You make a good argument against proposed feature (hadn't considered conflict w/ ctx menu on long press) -- you convinced me suggested feature is a bad idea. Thanks for taking the time to consider and reply!
